docs/content.zh/docs/ops/metrics.md

```diff
@@ -1328,7 +1328,7 @@ Note that the metrics are only available via reporters.
       <td>Histogram</td>
     </tr>
     <tr>
-      <th rowspan="20"><strong>Task</strong></th>
+      <th rowspan="22"><strong>Task</strong></th>
```

Comment:
Do we have a better place for those metrics? They are not very "task io" related

Reply:
There is the TaskMetricGroup which contains the TaskMetricIOGroup. However, the TaskMetricGroup seemed to me as the place used by the application developer to add custom metrics. Though I'm not sure about it.
If TaskMetricIOGroup is not the right place, it seems that we need to add new MetricGroup to TaskMetricGroup. What do you think?